File: lesson_4/other_langs_examples/python/example_cycle.py
ok_codes = [201,202,203,204,205]

# Вот этот код:
x = 0
while x < len(ok_codes):
	print(ok_codes[x])
	x += 1
	pass

"""
Аналогичен вот этому, на Go:

for x := 1; x < len(ok_codes); x++ {
	  fmt.Println(ok_codes[x])
	}
"""

# А вот этот:

for x in ok_codes:
	print(x)


"""
Вот этому коду на Go:

for _, value := range ok_codes{ 
	fmt.Println(value)}

"""

File: lesson_4/other_langs_examples/javascript/example_cycle.js
let ok_codes = [201,"202",203.0,false, null];

let i = 0;
while (i < (ok_codes.length-1)) {
  alert(ok_codes[i]);
  i++;
}

//Аналог на Go
/*
for x := 1; x < len(ok_codes); x++ {
	  fmt.Println(ok_codes[x])
	}
*/


for (let i = 0; (ok_codes.length-1); i++) {
  alert(ok_codes[i]);
}
//Аналог на Go
/*
for _, value := range ok_codes{ 
	fmt.Println(value)}
*/